src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -297,30 +303,36 @@ export default function ConsumerIncentivesHomeScreen() {
           </Trans>
         ) : hasMaxBalance ? (
           t('superchargeDisclaimerMaxRewards', { token: superchargingTokenConfig?.tokenSymbol })
+        ) : restrictSuperchargeForClaimOnly ? (
+          ''
```

Comment:
Shouldn't this condition be above hasMaxBalance first?

Reply:
the disclaimer for `hasMaxBalance` seems innocent enough which is why i didn't remove it, it currently doesn't encourage you to take any action (`"You’re getting maximum rewards on your {{token}}"`)